packages/codegen/generate.server.utils.js

```diff
@@ -641,6 +640,7 @@ function generateServerUtils (gqlOrSchemaName) {
         delete: delete_,
         softDelete,
         softDeleteMany,
+        hasFieldsParam: true, // TODO INFRA-538 remove this once migration to new server utils to be done
```

Comment:
What is this for?

Reply:
Ah, I see
Old utilities don't have this flag

Please write a comment about it

Reply:
This a temporary flag, just to notify developers - we have not migrated loadListByChunks utility to the new codegen utils.